CODIGO/frmOpciones.frm

```diff
@@ -1014,9 +1003,6 @@ Private Sub chkO_MouseUp(Index As Integer, Button As Integer, Shift As Integer,
             If AmbientalActivated = 1 Then
                 HScroll1.Enabled = False
                 AmbientalActivated = 0
```

Comment:
how do we stop active sounds?

Reply:
By calling
```
Public Function stop_wav(ByVal sound_id As Integer) As Long
Public Function stop_ambient() As Long
```

Music stops automatically when trying to play a new tune as we use only one track for now.

Reply:
what I mean here is that when the user change the setting the audio should stop playing, I don't understand how Automatically works here? are we checking the state of AmbientalActivated every frame?

Reply:
Ambient sounds loop, so they play constantly in background and we use a single track for them

Reply:
i think we need to call 
Call ao20audio.StopAmbientAudio
after setting AmbientalActivated = 0 as you did on BabelUI and for fx

Reply:
Added the call in the new patch